src/main/java/data/JsonConverter.java

```diff
@@ -51,4 +52,25 @@ public boolean updateFile(Vendor vendor) {
     createFile(vendor.getVendorID());
     return true;
   }
+
+  /** Retrieve and return the contents of the desired configuration. */
+  public String getConfig(String vendorID) {
+    String configContents = "";
+
+    // Retrieve the file with the corresponding vendorID.
+    File config = new File(FILE_PATH_BASE + vendorID);
+
+    Scanner input;
+    try {
+      input = new Scanner(config);
+    } catch(FileNotFoundException e) {
+      return null;
```

Comment:
This is fine for now, but in the future you may want to throw an exception here. Otherwise, the caller may do `String configString = getConfig(vendorId);` and think that the string will always be there. But then they do something with configString and get NullPointerException.

Reply:
A way to prevent what you said from happening is just to have a simple conditional if(configString == null) {// send a message to the frontend about what happened}. I understand what you are saying though.